### ISSUE TITLE
an error for "steps_per_epoch = display_interval"

### ISSUE THREAD

**Issue by nanzhao** [NTSC-Community/MatchZoo#50]
in main() function, there is a bug for display_interval parameter. It seems set wrongly in the main() funciton for train().
because in keras, steps_per_epoch doesn't mean it.

 for i_e in range(num_iters):
        for tag, generator in train_gen.items():
            genfun = generator.get_batch_generator()
            print('[%s]\t[Train:%s] ' % (time.strftime('%m-%d-%Y %H:%M:%S', time.localtime(time.time())), tag), end='')
            history = model.fit_generator(
                    genfun,
                    steps_per_epoch = display_interval,
                    epochs = 1,
                    shuffle=False,
                    verbose = 0
                ) #callbacks=[eval_map])

**Comment by yangliuy** [comment]
You can refer to our discussion on these parameters here: https://github.com/faneshion/MatchZoo/issues/40#issuecomment-360566397

**Comment by bwanglzu** [comment]
@nanzhao is correct, it's a bug.
